dart/lib/src/open_location_code.dart

```diff
@@ -327,9 +324,9 @@ String recoverNearest(
   var halfResolution = resolution / 2.0;
 
   // Use the reference location to pad the supplied short code and decode it.
-  var codeArea = decode(
-      encode(referenceLatitude, referenceLongitude)
-          .substring(0, paddingLength) + shortCode);
+  var codeArea = decode(encode(referenceLatitude, referenceLongitude)
```

Comment:
This formatting looks strange. Is this what dartfmt did?

Reply:
![image](https://user-images.githubusercontent.com/28839051/50142022-9ee84580-02b1-11e9-91f1-f9284852a8e9.png)